host/xtest/regression_4000.c

```diff
@@ -5766,4 +5864,135 @@ static void xtest_tee_test_4015(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 4015, xtest_tee_test_4015,
 		"Test TEE Internal API Derive key X25519");
+
+static void xtest_tee_test_4016_ed25519(ADBG_Case_t *c)
+{
+	TEEC_Session session = { };
+	TEE_OperationHandle op = TEE_HANDLE_NULL;
+	TEE_ObjectHandle key_handle = TEE_HANDLE_NULL;
+
+	TEE_Attribute key_attrs[2] = { };
+	size_t num_key_attrs = 0;
+	TEE_Attribute attrs[2] = { };
+	size_t num_attrs = 0;
+
+	uint8_t out[64] = { };
+	uint32_t out_size = sizeof(out);
+
+	size_t n = 0;
+	uint32_t ret_orig = 0;
+	size_t max_key_size = 0;
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+			xtest_teec_open_session(&session, &crypt_user_ta_uuid,
+			    NULL, &ret_orig)))
```

Comment:
Please align with the `(` on the line above. Same below where possible.

Reply:
done in [21de224](https://github.com/OP-TEE/optee_test/pull/610/commits/21de2241a42fc9ab579909deb55e0653a6faf5bf)